### DISCUSSION TITLE
Avatar component overlapping with Material UI Tabs

### DISCUSSION THREAD

**Discussion by jatinhemnani01** [chakra-ui/chakra-ui#4499] [Q&A]
How can I fix the overlapping of the component
![overlap](https://user-images.githubusercontent.com/73883835/127651174-493645e1-0721-45be-8440-5d6172eac75d.png)

**Comment by aaronkik** [comment]
Have you tried adding a z-index to the Tab component? 🤔

**Reply by jatinhemnani01** [reply]
what zindex should I give?

**Reply by aaronkik** [reply]
I would try a low number like 1 or 2 so it doesn't overlap any other components you may have.

**Reply by jatinhemnani01** [reply]
I fixed it by changing the position of Avatar to 'unset' and it now doesn't overlap.

Thanks for the help btw
